src/toast/tests/ops_groundfilter.py

```diff
@@ -0,0 +1,183 @@
+# Copyright (c) 2015-2020 by the parties listed in the AUTHORS file.
+# All rights reserved.  Use of this source code is governed by
+# a BSD-style license that can be found in the LICENSE file.
+
+import os
+
+import numpy as np
+import numpy.testing as nt
+
+from astropy import units as u
+from astropy.table import Column
+
+from .mpi import MPITestCase
+
+from ..noise import Noise
+
+from .. import ops as ops
+
+from ..vis import set_matplotlib_backend
+
+from ..pixels import PixelDistribution, PixelData
+
+from ._helpers import create_outdir, create_ground_data, fake_flags
+
+
+"""
+from .mpi import MPITestCase
+
+import os
+
+import numpy as np
+
+from ..tod import AnalyticNoise, OpSimNoise
+from ..todmap import OpGroundFilter, TODGround
+
+from ._helpers import create_outdir, create_distdata, boresight_focalplane
+"""
```

Comment:
Probably safe to delete this now, correct?

Reply:
Thanks!